src/core/Dates.js

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+
+/**
+ * Formats the date from UTC format into string
+ *
+ * @param  {object} with start and end numbers
+ * @return {string}
+ */
+const returnFormattedDate = (dates) => {
+    let startDate;
+    const endDate = moment(dates.end).format('Do MMM, YYYY');
+    /**
+     * Compares start and end date months
+     */
+    const startDateMonth = moment(dates.start).format('MM');
+    const endDateMonth = moment(dates.end).format('MM');
+    if (startDateMonth !== endDateMonth) {
```

Comment:
For the comparison here, you can use [Moment's Query methods](https://momentjs.com/docs/#/query/is-same/):

```
moment(dates.start).isSame(dates.end, 'month');
```